ios/FluentUI/Controls/SegmentedControl.swift

```diff
@@ -159,14 +186,19 @@ open class SegmentedControl: UIControl {
 
         return view
     }()
-    private var buttons = [SegmentedControlButton]()
+    private var buttons = [UIButton]()
     private let selectionView: UIView = {
         let view = UIView()
         view.layer.cornerCurve = .continuous
 
         return view
     }()
     private let bottomSeparator = Separator()
+    private let containerView: UIView = {
+        let view = UIView()
+        view.layer.cornerCurve = .continuous
```

Comment:
the layer will be curved regardless of tab or pill? or is this container only used for pill cases?

Reply:
if it is only for pill case maybe the property name should be  more specific

Reply:
The container view is only for the pill styles to add the insets. I'll update the name to be more specific.